src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final Random random = new Random();
+
+    public static Ball getRandomBall() {
```

Comment:
Didn't fix that: https://github.com/mate-academy/jv-lottery/pull/1441#discussion_r1549187968